src/qibolab/platforms/native.py

```diff
@@ -0,0 +1,200 @@
+from collections import defaultdict
+from dataclasses import dataclass, field
+from enum import Flag, auto
+from typing import Dict, List, Optional, Union
+
+from qibo import gates
+from qibo.config import raise_error
+
+from qibolab.pulses import FluxPulse, PulseConstructor, PulseSequence, PulseType
+
+
+@dataclass
+class NativePulse:
+    """Container with parameters required to generate a pulse implementing a native gate."""
+
+    name: str
+    duration: int
+    amplitude: float
+    shape: str
+    pulse_type: PulseType
+    qubit: "Qubit"
+    frequency: int = 0
+    relative_start: int = 0
+    """Relative start is relevant for two-qubit gate operations which correspond to a pulse sequence."""
+
+    # TODO: Remove these attributes if they are not useful, as they are not used anywhere
+    if_frequency: int = 0
```

Comment:
This is needed for qblox I believe

Reply:
That's not a problem, we can leave it here, I just updated the comment.